httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/MultipartEntityBuilder.java

```diff
@@ -47,18 +47,8 @@
 /**
  * Builder for multipart {@link HttpEntity}s.
  * <p>
- * This class constructs multipart entities with a boundary determined by either a fixed
- * value ("httpclient_boundary_7k9p2m4x8n5j3q6t1r0vwyzabcdefghi") or a random UUID. If no
- * boundary is explicitly set via {@link #setBoundary(String)}, it defaults to the fixed
- * value unless {@link #withRandomBoundary()} is called to request a random UUID at build
- * time. Users can provide a custom boundary with {@link #setBoundary(String)}. A warning
- * is logged when no explicit boundary is set via {@link #setBoundary(String)}, encouraging
- * deliberate choice.
- * </p>
- * <p>
- * IMPORTANT: it is responsibility of the caller to validate / sanitize content of body
```

Comment:
@benweissmann Restore this section. It says IMPORTANT for a reason.

Reply:
Done -- i had added a similar `IMPORTANT` note in `setBoundary` to note the considerations specifically around boundary values, but happy to leave it here as well.